client/components/comment.jsx

```diff
@@ -96,14 +129,16 @@ export default class Comment extends React.Component {
                     </div>
                 </form>    
                     <div className="com-createButtons">
-                        <button className="form-buttons" title={`Like`}>
-                            <i className="fas fa-heart"></i>Like</button>
-                        <button className="form-buttons" title={`Repost`}>
-                            <i className="fas fa-retweet"></i>Repost</button>
-                        <button className="form-buttons" title={`Share`}>
-                            <i className="fas fa-share-square"></i>Share</button>
-                        <button className="form-buttons" title={`More Options`}>
-                            <i className="fas fa-ellipsis-h"></i>More</button>
+                        <button className={`com-button com-${this.state.like}`} title={`Like`} onClick={this.toggleLike}>
```

Comment:
For this part I would make this a few arrays:
const titles = ['Like', 'Repost', ... ]
const faSymbols = ['heart', 'retweet', ... ] 

And map it all before the render.
const buttons = [];
titles.forEach((title, i) => <button className={...} ... >);

This way you don't have to copy and paste this text